README.md

```diff
@@ -294,6 +294,11 @@ All previous rules can be added at the same time by just adding the BaristaRule.
 @Rule
 public BaristaRule<MyActivity> baristaRule = BaristaRule.create(MyActivity.class);
 
+or kotlin way...
```

Comment:
Hmm we don't yet announce or officially support Kotlin APIs. 
Adding this to the readme is a declaration of intent that I'm not sure we want to make... 🤔